src/data_request.rs

```diff
@@ -13,6 +13,7 @@
 pub struct DataRequest {
     pub layers: bool,
     pub lib: bool,
+    pub glyph_lib: bool,
```

Comment:
Is this a useful addition, or is it only useful in the context of benchmarking?

Reply:
This was primarily added to test the impact of glyphlib loading on loading times. Result: it roughly doubles parsing time on my Linux and Windows machines. Ouch.